### PR TITLE
fix: Oriented point rendering in Neuroglancer

### PR DESCRIPTION
**Pull request by manasaV3**
Relates to https://github.com/chanzuckerberg/cryoet-data-portal/issues/1059

## Description

- Fixes the rendering of oriented point in neuroglancer.
- Adds tests for viz config generation

### CODE REVIEW

File: ingestion_tools/scripts/importers/visualization_config.py [hunk 1]
```diff
@@ -90,18 +90,21 @@ def _to_point_layer(
...
-            f"{name_prefix} point",
-            self.config.https_prefix,
-            color=color,
-            scale=resolution,
-            is_visible=file_metadata.get("is_visualization_default"),
-            is_instance_segmentation=is_instance_segmentation,
-        )
+        args = {
+            "source": source_path,
+            "name": f"{name_prefix} point",
```
Comment (manasaV3) [inline ingestion_tools/scripts/importers/visualization_config.py:98]: @uermel  As the point and the orientend point are going to be rendered differently, should we distinguish that in the name as well?

File: ingestion_tools/scripts/importers/visualization_config.py [hunk 2]
```diff
@@ -90,18 +90,21 @@ def _to_point_layer(
...
+            "url": self.config.https_prefix,
+            "color": color,
+            "scale": resolution,
+            "is_visible": file_metadata.get("is_visualization_default"),
+            "is_instance_segmentation": shape == "InstanceSegmentation",
```
Comment (jgadling) [inline ingestion_tools/scripts/importers/visualization_config.py:103]: nit: not sure if it's just me, but this is kindof ugly imop.

File: ingestion_tools/scripts/importers/visualization_config.py (hunk 1 above)
Comment (uermel) [reply ingestion_tools/scripts/importers/visualization_config.py:98]: I agree conceptually, but would the names become too long if we add 'oriented'?

**Review by jgadling — APPROVED** [review]

File: ingestion_tools/scripts/importers/visualization_config.py (hunk 1 above)
Comment (uermel) [reply ingestion_tools/scripts/importers/visualization_config.py:98]: On second thought 

`segmentation`
`orientedpoint`

have the same length, so I agree we should name properly.

**Review by uermel — APPROVED** [review]

File: ingestion_tools/scripts/importers/visualization_config.py (hunk 2 above)
Comment (manasaV3) [reply ingestion_tools/scripts/importers/visualization_config.py:102]: The variable name or doing the check against the shape value?

Comment (jgadling) [reply ingestion_tools/scripts/importers/visualization_config.py:102]: setting the value for the `is_instance_segmentation` key with an inline boolean expression

**Pull request merged**